README.md

```diff
@@ -1,5 +1,7 @@
 [![tests](https://github.com/alex-petrenko/sample-factory/actions/workflows/test-ci.yml/badge.svg?branch=sf2)](https://github.com/alex-petrenko/sample-factory/actions/workflows/test-ci.yml)
 [![codecov](https://codecov.io/gh/alex-petrenko/sample-factory/branch/sf2/graph/badge.svg?token=9EHMIU5WYV)](https://codecov.io/gh/alex-petrenko/sample-factory)
+[![pre-commit](https://github.com/wmFrank/sample-factory/actions/workflows/pre-commit.yml/badge.svg?branch=sf2-pre-commit)](https://github.com/wmFrank/sample-factory/actions/workflows/pre-commit.yml)
+[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/wmFrank/sample-factory/master.svg)](https://results.pre-commit.ci/latest/github/wmFrank/sample-factory/master)
```

Comment:
Need to update the badges?

Reply:
Sure, and 'pre-commit.ci' status only works on main/master branch. So we can use that when we push sf2 to master in the future.